.travis.yml

```diff
@@ -46,7 +43,7 @@ matrix:
           env: CONFIG_OPTS="--debug --coverage no-asm enable-rc5 enable-md2 enable-ec_nistp_64_gcc_128 enable-ssl3 enable-ssl3-method enable-nextprotoneg enable-weak-ssl-ciphers enable-external-tests no-shared -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION" COVERALLS="yes" BORINGSSL_TESTS="yes" CXX="g++-5"
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
```

Comment:
Well, provided that one of things memory sanitizer does is tracing uninitialized memory, all assembly subroutines' output will be considered uninitialized. And this is platform-neutral thing. So I suppose question is reduced to just why don't other sanitizers get same treatment. With background that they remain blind to assembly that is...